packages/tuono-router/src/hooks/useRouter.ts

```diff
@@ -13,7 +15,13 @@ interface UseRouterResult {
   /**
    * Redirects to the path passed as argument updating the browser history.
    */
-  push: (path: string, opt?: PushOptions) => void
+  push: (path: string, opt?: NavigateOptions) => void
```

Comment:
Since the name of the type is `NavigateOptions` I would consider to rename all `opt` (singular) to `opts`, just for consistency